src/zkevm_specs/tx.py

```diff
@@ -337,18 +338,19 @@ def tx2witness(
 
     sign_verification = SignVerifyChip.assign(sig, pk, tx_sign_hash, randomness)
 
+    call_data_gas_cost = sum([(4 if byte == 0 else 16) for byte in tx.data])
```

Comment:
We have `GAS_COST_TX_CALL_DATA_PER_ZERO_BYTE` and `GAS_COST_TX_CALL_DATA_PER_NON_ZERO_BYTE` in `param.py`